src/xrpld/app/tx/detail/PermissionedDomainSet.cpp

```diff
@@ -30,8 +30,10 @@ namespace ripple {
 NotTEC
 PermissionedDomainSet::preflight(PreflightContext const& ctx)
 {
-    if (!ctx.rules.enabled(featurePermissionedDomains))
+    if (!ctx.rules.enabled(featurePermissionedDomains) ||
```

Comment:
This change require amendment as PermissionedDomain already released. Old call will return `tesSuccess` and new one `temDISABLED`

Reply:
PermissionedDomain hasn't been released yet, just merged.

Reply:
But how it is already in the `release` branch?

Reply:
Because it's in a beta, so it's on Devnet but not on mainnet yet. And all the nodes on Devnet are always running the same version, so discrepancies aren't important there.

Reply:
As @mvadari is saying. Unless we merge this **after** 2.4.0 is actually released (it's not, as of today), there is no need for an additional amendment. Unreleased code is **not** part of the protocol, and beta (or release candidate) releases do not count.